.codecov.yml

```diff
@@ -1,4 +1,19 @@
 coverage:
   status:
-    project: no
+    project:
+      default:
+        target: 75%
```

Comment:
Will this be enforced by github? If so, why isn't there any check on this PR visible?

Reply:
According to https://docs.codecov.io/docs/flags it does. But we currently have
```
Flags have been temporarily removed from this view while the flagging feature is refactored for better performance and user experience. 
```
on https://codecov.io/gh/raiden-network/raiden/pull/4547, which might influence such things. I'll rebase this PR and see what the current result looks like.

Reply:
I now see a check `codecov/project — 81.91% (+1.09%) compared to a5ade15 ` on github. I've also seen it red before all coverages have been collected. So it seems to work.